src/pages/admin/home.tsx

```diff
@@ -15,59 +15,34 @@ type Props = {
     initialData: BiztechEvent[] | null
 }
 
-// type definition for a BiztechEvent
-type BiztechEvent = {
-    id: string;
-    year: number;
-    capac: number;
-    createdAt: number;
-    description: string;
-    elocation: string;
-    ename: string;
-    startDate: string;
-    endDate: string;
-    imageUrl: string;
-    updatedAt: number;
-};
-
 export default function AdminEventView({ initialData }: Props) {
     const router = useRouter()
-    // state for if data is still loading or not
     const [isLoading, setLoading] = useState(!initialData)
-    // data represents the BiztechEvent list
     const [data, setData] = useState<BiztechEvent[] | null>(initialData);
-    // add a mobile state and change grid to block if mobile view
     const [isMobileDevice, setIsMobileDevice] = useState(false);
-    // isClicked state for when the "more" icon is clicked (mobile only)
     const [isClicked, setIsClicked] = useState(false);
-    // isDelete state for when the "delete event" button is pressed
     const [isDelete, setIsDelete] = useState(false)
-    // bizEvent is a state to know which event has been clicked
-    const [bizEvent, setBizEvent] = useState("Event Name")
+    const [bizEvent, setBizEvent] = useState<BiztechEvent | null>()
 
 
     // toggle the view when the 'more' icon is clicked 
-    const clickEffect = (eventClicked?: string) => {
-        if (eventClicked)
-            setBizEvent(eventClicked) // allows us to grab the specific event which was clicked
+    const eventClick = (event: BiztechEvent) => {
+        if (event)
+            setBizEvent(event) // allows us to grab the specific event which was clicked
```

Comment:
If we're not using brackets after an `if`, it would be better to have these on the same line to avoid any confusion. Otherwise, consider adding brackets